app/api/drf/views.py

```diff
@@ -0,0 +1,106 @@
+from django.contrib.auth.models import User
+from rest_framework import viewsets
+from rest_framework.decorators import api_view
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from app.api.drf.serializers import CreateUserSerializer, UserSerializer, BeerSerializer, BrewPiDeviceSerializer, FermentationProfileSerializer, FermentationProfilePointSerializer, BeerLogPointSerializer
+from app.models import Beer, BrewPiDevice, FermentationProfile, FermentationProfilePoint, BeerLogPoint
+from rest_framework import generics
+
+class CreateUserViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows users to be created
+    """
+    permission_classes = [AllowAny,]
+    queryset = User.objects.all().order_by('-date_joined')
+    serializer_class = CreateUserSerializer
+    http_method_names = ['post']
+
+    def post(self, request, format=None):
+        serializer = UserSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+class UserViewSet(viewsets.ModelViewSet):
```

Comment:
Similarly, what's the thought with exposing the full user list to all users? How would this be used by an external app?